source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1348,8 +1352,16 @@ def _tableHeaderTextHelper(self, axis):
 			# Each header must be fetched from the headers array once and only once,
 			# as it gets released when it gets garbage collected.
 			for i in range(nHeaders):
+				header = headers[i]
+				# Chromium exposes cells as their own headers, so exclude cells with the same `Iaccessible2::uniqueID`.
+				if cellIA2UID is not None:  # No point checking if we don't have this cell's UID
+					try:
+						if cellIA2UID == header.QueryInterface(IA2.IAccessible2).uniqueID:
+							continue
+					except COMError:
+						pass  # Assume different UIDs for safety
 				try:
-					text = headers[i].QueryInterface(IA2.IAccessible2).accName(0)
+					text = header.QueryInterface(IA2.IAccessible2).accName(0)

```

Comment:
It would be good to avoid queryInterface to IAccessible2 twice. Probably best to queryInterface to IAccessible2 on the header much further up, so that fetching of uniqueID and accName can be done on the same one.